concourse/pipelines/create-microbosh.yml

```diff
@@ -225,7 +240,8 @@ jobs:
 
             ./build_manifest.sh \
               ../../../bosh-secrets/bosh-secrets.yml \
-              ../../../terraform-outputs/terraform-outputs.yml \
+              ../../../terraform-outputs/bosh.terraform-outputs.yml \
```

Comment:
Yes, I wasn't aware that we've got two scripts with the same name to spruce manifests but in different directories and working in a different way. It would be good to have just one that we can parametrise. I'll put this as a suggestion in [Pipelines improvements](https://www.pivotaltracker.com/n/projects/1275640/stories/111700216).